### PR TITLE
Feature 6

### PR DESCRIPTION
**Pull request by Namrata-J**
#7 added image and input component.
Please review it, thankyou!!

### CODE REVIEW

File: components/images/responsive-images.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+/* simple responsive image start here */
+.et_simple-ri{
+    width: 80%;
+}
+.et_simple-ri img{
+    vertical-align:bottom;
+    max-width: 60%;
+}
+/* simple responsive image end here */
```
Comment (Raghav888) [inline components/images/responsive-images.css:30]: You can add code for responsiveness given by vikram

File: components/images/responsive-images.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+/* rounded responsive image start here */
+.et_rounded-ri{
+    padding: 1rem;
+}
+.et_rounded-ri img{
+    clip-path: circle();
+    min-width: 50px;
```
Comment (Raghav888) [inline components/images/responsive-images.css]: Please use rem

**Review by Raghav888 — APPROVED** [review]
Approved

**Review by dev-enforced — APPROVED** [review]
You can remove the inputs.js file mentioned in the inputs folder if it isn't required. Rest of the code looks good to me. Good work on the Images and Input component Namrata.

File: components/inputs/inputs.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
+@import url(/css/base.css);
+@import url(/css/utility.css);
+
+/* input component start here */
+.et_input-container{
+align-items: center;
+display: flex;
+flex-direction: column;
+justify-content: center;
+}
```
Comment (ozairkhan2810) [inline components/inputs/inputs.css]: use prettier to increase readability

File: components/images/responsive-images.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    max-width: 60%;
+}
+/* simple responsive image end here */
+/* rounded responsive image start here */
+.et_rounded-ri{
+    padding: 1rem;
+}
+.et_rounded-ri img{
+    clip-path: circle();
+    min-width: 3.125rem;
```
Comment (ozairkhan2810) [inline components/images/responsive-images.css]: the value is odd, just round off

File: components/images/responsive-images.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+.et_ri-container>*{
+    background-color: var(--white-color);
+    border: 1px solid var(--light-gray);
+    margin: 2rem;
+    padding: 2rem 0rem;
```
Comment (ozairkhan2810) [inline components/images/responsive-images.css]: 0 instead of 0rem

**Review by ozairkhan2810 — APPROVED** [review]
small changes, can be merged

File: components/images/responsive-images.css (hunk 2 above)
Comment (Namrata-J) [reply components/images/responsive-images.css]: > You can remove the inputs.js file mentioned in the inputs folder if it isn't required. Rest of the code looks good to me. Good work on the Images and Input component Namrata.

yes, will remove it. Thankyou so much for reviewing!!

**Comment by Namrata-J** [comment]
> You can remove the inputs.js file mentioned in the inputs folder if it isn't required. Rest of the code looks good to me. Good work on the Images and Input component Namrata.

yes, will remove it. Thankyou so much for reviewing!!

File: components/inputs/inputs.css (hunk 3 above)
Comment (Namrata-J) [reply components/inputs/inputs.css]: > small changes, can be merged

hey ozair, thanks for the suggestions, will certainly do it! Thankyou so much for reviewing!!

**Pull request merged**